packages/o-spreadsheet-engine/src/functions/module_lookup.ts

```diff
@@ -1104,3 +1104,193 @@ export const OFFSET = {
     );
   },
 } satisfies AddFunctionDescription;
+
+// -----------------------------------------------------------------------------
+// CHOOSE
+// -----------------------------------------------------------------------------
+export const CHOOSE = {
+  description: _t("An element from a list of choices based on index."),
```

Comment:
Probably missing a word in the description. `Chose an element` ? `Return an element` ?